src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1050,23 +1169,6 @@ public bool IsTopMostNode
             }
         }
 
-        internal void ConnectInput(int inputData, int outputData, NodeModel node)
```

Comment:
@gregmarr @mjkkirschner @ramramps I hate to ruin what is turning into a very interesting conversation, but I'm rolling back the changes to remove `ConnectInput` and `ConnectOutput`, and the changes to the properties. They were, I thought, a nice cleanup while I was in there, but they are orthogonal to the purpose of this PR. We can revisit this conversation in the future.